docs/content/integrations/bigquery/using-bigquery-with-dagster.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: "Using Dagster with BigQuery | Dagster Docs"
+description: Store your Dagster assets in BigQuery
+---
+
+# Using Dagster with BigQuery
+
+This tutorial focuses on how to store and load Dagster's [software-defined assets (SDAs)](/concepts/assets/software-defined-assets) in BigQuery.
+
+By the end of the tutorial, you will:
+
+- Configure a BigQuery I/O manager
+- Create a table in BigQuery using a Dagster asset
+- Make a BigQuery table available in Dagster
+- Load BigQuery tables in downstream assets
+
+This guide focuses on storing and loading Pandas DataFrames in BigQuery. Dagster also supports using PySpark DataFrames with BigQuery. The concepts from this guide apply to working with PySpark DataFrames, and you can learn more about setting up and using the BigQuery I/O manager with PySpark DataFrames in the [reference guide](/integrations/bigquery/reference).
+
+---
+
+## Prerequisites
+
+To complete this tutorial, you'll need:
+
+- **To install the `dagster-gcp` and `dagster-gcp-pandas` libraries**:
+
+  ```shell
+  pip install dagster-gcp dagster-gcp-pandas
+  ```
+
+- **To gather the following information**, which is required to use the BigQuery I/O manager:
```

Comment:
The colon here makes me think there's supposed be something that follows this - is something missing?